tracing-error/src/lib.rs

```diff
@@ -196,98 +200,23 @@
     while_true
 )]
 mod backtrace;
-#[cfg(not(feature = "stack-error"))]
-mod heap_error;
+#[cfg(feature = "traced-error")]
+mod error;
 mod layer;
-#[cfg(feature = "stack-error")]
-mod stack_error;
 
 pub use self::backtrace::{SpanTrace, SpanTraceStatus};
-#[cfg(not(feature = "stack-error"))]
-pub use self::heap_error::TracedError;
+#[cfg(feature = "traced-error")]
```

Comment:
I think this needs a `doc-cfg` attribute.

Reply:
I tried putting it here first and it had no effect so I moved it to the definitions of the structs / traits in error.rs to get it to actually show up in the docs